src/Aspire.Hosting.Azure.AppContainers/ContainerAppEnvironmentDotnetComponentResource.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Provisioning.Primitives;
+
+namespace Azure.Provisioning.AppContainers;
+
+// This is missing from Azure.Provisioning.AppContainers
+internal sealed class ContainerAppEnvironmentDotnetComponentResource(string bicepIdentifier, string? resourceVersion = null) : ProvisionableResource(bicepIdentifier, new("Microsoft.App/managedEnvironments/dotNetComponents"), resourceVersion)
```

Comment:
When do we get the real provisioning types from Azure SDK?

Reply:
@tg-msft